pyproject.toml

```diff
@@ -147,10 +147,11 @@ docs = [
     "sphinx-gallery",
     "sphinx_rtd_theme",
     "numpydoc",
-    "MEArec",
-    "datalad==0.16.2",
-    "pandas", # Don't know why yet
-    "hdbscan",   # For sorters I think
+    "MEArec",   # For examples
+    "datalad==0.16.2",  # Download mearec data in examples
+    "pandas", # Don't know where this is needed
+    "hdbscan",   # For sorters, probably spikingcircus
+    "numba", # For sorters, probably spikingcircus
     "neo @ git+https://github.com/NeuralEnsemble/python-neo.git",  # Until next release
```

Comment:
@h-mayorquin for the docs, since it builds for every PR, I would always install both NEO and probeinterface from source. What do you think?

Reply:
Not doing NEO and probeinterface from source broke a bunch of examples in the past so I would second this.

Reply:
This makes sense to me. Changed.